### DISCUSSION TITLE
Can I use the Prisma Plugin with multiple prisma db connections?

### DISCUSSION THREAD

**Discussion by dylanlarrivee** [hayes/pothos#1169] [Q&A] [answered]
I am using prisma to connect to 2 different dbs using 3 seperate `schema.prisma` files. I am wondering if there is a way to use the plugin with all 3 prisma connections or if I am only able to use with 1 and should just use it on my main db.

**Comment by hayes** [comment, marked as answer]
Currently the prisma plugin only supports a single DB schema.  So you'll have to stick to using it for your main schema.   Fortunately, creating prisma based types in Pothos without a plugin is still pretty easy

**Reply by hayes** [reply]
The only way around this currently is to define multiple GraphQL schemas with separate schema builders and then merge the schemas together

**Reply by dylanlarrivee** [reply]
Ok cool, yea I saw [this](https://pothos-graphql.dev/docs/plugins/prisma#using-prisma-without-a-plugin) in the documentation that shows how to use prisma without the plugin and it looks like a pretty easy setup.

Is it correct to say that this setup works as is without a dataloader feature in Pothos because it uses the built in dataloader that Prisma uses?

**Reply by hayes** [reply]
more or less yes.  The prisma dataloader will work well for the simple cases.  It's easy to end up accidentally writing something that causes the prisma dataloader to not handle the n+1 case correctly, so using an explicit dataloader might be safer in some situations.  I would probably just see how its working for you, and if needed you can always add dataloaders later

**Reply by dylanlarrivee** [reply]
Awesome, thanks so much for the info.
